integrations-and-supported-tools/pytorch-lightning/scripts/Neptune_Pytorch_Lightning_more_options.py

```diff
@@ -188,7 +177,7 @@ def log_confusion_matrix(lit_model, data_module):
     test_data = data_module.test_dataloader()
     y_true = np.array([])
     y_pred = np.array([])
-    for i, (x, y) in enumerate(test_data):
+    for x, y in test_data:
```

Comment:
Let's do not mixin changes that are outside the scope of the PR. We can keep this as an exception.